### PR TITLE
Simple Linux App Tutorial!

### PR DESCRIPTION
**Pull request by Ry10p**
My First Tutorial Ever!!!

### CODE REVIEW

File: docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world! [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    def main():
+        # This needs to return an object that has a main_loop() method.
+        return None
+
+This won't do much as it is, but we can make it useful. 
+
+Add this into the ``app.py`` to make it useful:
+
+.. code-block:: python
+    import __main__
```
Comment (freakboy3742) [inline docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world!]: Ok - this line shouldn't be needed. Why have you added it?

File: docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world! [hunk 2]
```diff
@@ -0,0 +1,84 @@
+Linux Ubuntu 16.04 Tutorial 0 - Hello, world!
+==========================
```
Comment (freakboy3742) [inline docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world!]: ReST formatting requires that the underlining is the same length as the text you are underlining - so you need a few more `=` here. 

If you want to check the formatting, you can run `make html` in the docs directory. You may need to `pip install sphinx` first.

File: docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world! [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+In this tutorial, you'll take a really simple "Hello, world!" program written in
+Python, convert it into a working Ubuntu Linux project.
```
Comment (freakboy3742) [inline docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world!]: Is there anything completely Ubuntu specific here? AFAIK, this set of instructions should work for *any* Linux; so I'd rather label this as a "Linux" tutorial, with a note early on that says "this has been tested on Ubuntu 16.04, but should work on other Unixes"

File: docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world! [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+Setup
+-----
+
+This tutorial assumes you've read and followed the instructions in
+:doc:`/intro/getting-started`. This Tutorial is for Linux Users developing for Ubuntu 16.04!
+* A ``tutorial`` directory,
+* A activated Python 3 virtual environment,
```
Comment (freakboy3742) [inline docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world!]: Plural - "An activated"

File: docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world! [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+There is a new folder in your project called 'linux', which contains the project (tutorial_0). You can open the application by running:
+
+.. code-block:: bash
+
+    ./linux/tutorial_0
+
+The app should run like this!
+.. code-block:: bash
+
+    (BeeWare) ry@X1Carbon:~/Native/tutorial_0$ ./linux/tutorial_0
```
Comment (freakboy3742) [inline docs/tutorials/Linux Ubuntu 16.04 Tutorial 0 - Hello, world!]: You can drop the prefix here, and just use the `$` prompt

**Comment by Ry10p** [comment]
Added a new pull request

**Pull request closed without merging**